pyproject.toml

```diff
@@ -1,31 +1,28 @@
-[tool.poetry]
+[project]
 name = "python-fastapi-template"
 version = "0.1.0"
-description = "A template for a Python FastAPI service with Poetry & Devcontainer"
+description = "A template for a Python FastAPI service with uv & Devcontainer"
 authors = ["Ibraheem Tuffaha <ibraheem.z.tuffaha@gmail.com>"]
 license = "MIT"
 readme = "README.md"
-package-mode = false
-
-[tool.poetry.dependencies]
-python = "3.12.x"
-fastapi = "^0.111.0"
-
-
-[tool.poetry.group.dev.dependencies]
-pre-commit = "^3.7.1"
-black = "^24.4.2"
-isort = "^5.13.2"
-mypy = "^1.10.1"
-flake8 = "^7.1.0"
-autoflake = "^2.3.1"
-wemake-python-styleguide = "^0.19.2"
-pytest = "^8.2.2"
-pytest-cov = "^5.0.0"
-
-[build-system]
-requires = ["poetry-core"]
-build-backend = "poetry.core.masonry.api"
+requires-python = ">=3.12"
+dependencies = [
+    "fastapi[standard]>=0.115.8",
+]
+
+
+[dependency-groups]
+dev = [
```

Comment:
I don't know if this should be changed to:

```
[tool.uv]
dev-dependencies = [
]
```

Reply:
I added these using `uv add [DEPENDENCY] --dev` like recommended in docs
https://docs.astral.sh/uv/concepts/projects/dependencies/#development-dependencies